cypress/jenkins/cypress.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -x
+set -e
+
+pwd
+ls -al
+cd "dashboard"
+
+yarn install
+yarn add --dev -W mocha mochawesome mochawesome-merge \
+  mochawesome-report-generator cypress-multi-reporters \
+  mocha-junit-reporter 
+
+NO_COLOR=1 cypress run --browser chrome \
```

Comment:
There's a few args missing here

`TEST_BASE_URL` - points to the Rancher instance tests will run on
`TEST_USERNAME` (defaults to admin) - user to run the test as
`TEST_PASSWORD` - password of the user to run the tests as.
Depending on the Rancher install you also might need

`CATTLE_BOOTSTRAP_PASSWORD` - bootstrap password that will be used to setup rancher